### PR TITLE
Activate OCP quickstart link on website

### PR DESCRIPTION
**Pull request by raphink**
image

### CODE REVIEW

File: site/data/en/homepage.yml [hunk 1]
```diff
@@ -25,9 +25,9 @@ banner:
         #- label: "SKS Quickstart"
         #  icon: "fas fa-mountain"
         #  link: "/docs/devops-stack/__VERSION__/howtos/quickstart_sks.html"
-        #- label: "OpenShift Quickstart"
-        #  icon: "fab fa-redhat"
-        #  link: "/docs/devops-stack/__VERSION__/howtos/quickstart_openshift.html"
+      - label: "OpenShift Quickstart"
```
Comment (mcanevet) [inline site/data/en/homepage.yml:28]: Maybe we should specify that it is OpenShift on AWS only.

**Review by mcanevet — COMMENTED** [review]
Maybe we should specify that it is OpenShift on AWS only

**Comment by raphink** [comment]
@mcanevet that will be a pretty large button though

**Comment by mcanevet** [comment]
Indeed

**Review by mcanevet — APPROVED** [review]

**Pull request merged**